app/views/layouts/application.html.haml

```diff
@@ -6,12 +6,14 @@
     %meta{content: "width=device-width, initial-scale=1, shrink-to-fit=no", name: "viewport"}
     %meta{content: "Festive, Rugged, Unique Texas Trail Races", name: "description"}
     %meta{content: "Spectrum Trail Racing", name: "author"}
+    %csrf_meta_tags
     %title Spectrum Trail Racing
     %link{:href => "https://fonts.googleapis.com/css?family=Nanum+Pen+Script|Nunito:400,600,700,800|Roboto:400,500,700&display=swap", :rel => "stylesheet"}/
     = Gon::Base.render_data
     = favicon_link_tag asset_path("favicon")
     = stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload"
     = javascript_include_tag "application", "data-turbolinks-track": "reload"
+    = csrf_meta_tags
```

Comment:
Do I need this line in the devise.html.haml or mailer.html.haml layout files? I tried it but still got the protect_against_forgery error.